Correct a class name JLoggerCallback => JLogLoggerCallback

elkuku opened the pull request:
I believe the class name is wrong.

elkuku committed:
Correct a class name JLoggerCallback => JLogLoggerCallback

I believe the class name is wrong.

LouisLandry commented:
You are correct, but this kills the unit tests.

`PHP Fatal error:  Class 'JLoggerCallback' not found in /var/lib/jenkins/workspace/pull-requests/tests/suites/unit/joomla/log/loggers/stubs/callback/inspector.php on line 23`

`require_once() /var/lib/jenkins/workspace/pull-requests/tests/suites/unit/joomla/log/loggers/JLogLoggerCallbackTest.php:11`

If you can fix that up I'll get this merged.  Thanks a bunch Nikolai!

_I'm going to close this until it's fixed.  Please leave a comment to reopen it when you have updated the branch and we'll reopen it and get it merged._

elkuku committed:
Fix unit tests due to class renaming

JLoggerCallback => JLogLoggerCallback

elkuku commented:
Should be good to reopen now.

elkuku commented:
erm.. reopen ? re-open ? **REOPEN** ... ¿

pasamio commented:
You should probably fix the class name of the inspector as well if we're going to be renaming things to match.

elkuku committed:
Fix unit tests class names

JLoggerCallback => JLogLoggerCallback
JLoggerCallbackInspector => JLogLoggerCallbackInspector
JLoggerCallbackTestHelper => JLogLoggerCallbackTestHelper
jLoggerCallbackTestHelperFunction => jLogLoggerCallbackTestHelperFunction

elkuku commented:
OK, I changed a few class names and function names and the tests are still running on my end..

LouisLandry commented on tests/suites/unit/joomla/log/loggers/JLogLoggerCallbackTest.php:
```diff
@@ -7,21 +7,21 @@
  */
 
-require_once JPATH_PLATFORM . '/joomla/log/logger/callback.php';
+//require_once JPATH_PLATFORM . '/joomla/log/logger/callback.php';
```
Nitpicky I know, but would you just remove this line instead of commenting it out.  Everything looks like it passes so we will get it merged when you clear that out.  Thanks a bunch!

elkuku commented:
I hope we are done now ;)

LouisLandry commented:
LOL, thanks for your patience @elkuku ... I really appreciate it. :+1:
